.circleci/config.yml

```diff
@@ -112,14 +112,47 @@ jobs:
       - store_artifacts:
           path: /root/sd/junit
 
+  tests:
+    machine: true
+    environment:
+      DOCKER_API_VERSION=1.23
+    parallelism: 3
+    steps:
+      - run: sudo apt-get install -y make
+      - checkout
+
+      - run:
+          name: Build Docker images
+          command: |
+            set +o pipefail
```

Comment:
Why are you adding a `+o pipefail`? That's the default, and I'm pretty sure we *want* the error code to propagate here to abort if an earlier command fails.

Reply:
the default is '-eo pipefail', the intent here is to prevent cache population failing from interrupting the build.